### DISCUSSION TITLE
Does `reset` function persist reference between rerenders?

### DISCUSSION THREAD

**Discussion by valerii15298** [react-hook-form/react-hook-form#8276] [Q&A] [answered]
I am using it like this:
```tsx
  useEffect(() => {
    reset(defaultValues);
  }, [defaultValues, reset]);
```
Can I include `reset` into dependency array safely? Can I be sure it will not change reference to itself?
I know I can just omit it from deps array. But I am curious if it is safe to include it(eslint is complaining when I omit it)

**Comment by bluebill1049** [comment, marked as answer]
yes
